mypy_whitelist.txt

```diff
@@ -0,0 +1,17 @@
+pandas/core/dtypes/base.py
+pandas/core/groupby/groupby.py
+pandas/core/internals/blocks.py
+pandas/core/internals/managers.py
+pandas/core/common.py
```

Comment:
why is this not possible in the setup file?

Reply:
Yea I was hoping that `Mypy_path` in the ini file would work but didn't have any luck. I didn't also see a very good way to make this readable therein since the ini would expect this to be a comma separated list.

With that said this may just be temporary anyway - would ideally like to run on the entire code. Modules without existing types would be ignored by default anyway